transformations/contextual_meaning_perturbation/transformation.py

```diff
@@ -0,0 +1,186 @@
+import itertools
+import logging
+import random
+from collections import defaultdict
+
+import spacy
+from transformers import pipeline
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
+class ContextualMeaningPerturbation(SentenceOperation):
+    tasks = [
+        TaskType.TEXT_CLASSIFICATION,
+        TaskType.QUALITY_ESTIMATION,
+    ]
+    languages = ["en", "de"]
+    heavy = True
+
```

Comment:
Could you please add keywords for this transformation?

Check [available keywords](https://github.com/GEM-benchmark/NL-Augmenter/blob/main/docs/keywords.md) and [this example](https://github.com/GEM-benchmark/NL-Augmenter/blob/84db89bce28169bec1eaad240cb70d3d89f4c344/transformations/butter_fingers_perturbation/transformation.py#L82)